Spectrum/BetaQuest.cs

```diff
@@ -788,13 +810,19 @@ private static void LocateBetaQuestEntrance(Arguments args)
 
             short index = (short)args[0];
             short[] VAR_Entrances = BetaQuest.GetVAR_Entrances();
+            Dictionary<short, string> Entrance_Table = BetaQuest.GetEntrance_Table();
+            Dictionary<short, string> Grotto_Table = BetaQuest.GetGrotto_Table();
 
-            for (int i = 0; i < 326; i++)
+            for (int i = 0; i < VAR_Entrances.Length; i++)
             {
-                var read = shuffleTable.ReadInt16(i * 2);
+                short read = shuffleTable.ReadInt16(i * 2);
                 if (index == read)
                 {
-                    Console.WriteLine($"{VAR_Entrances[i]:X4}");
+                    short entranceId = VAR_Entrances[i];
+                    string fromEntrance = Grotto_Table.ContainsKey(entranceId) ? Grotto_Table[entranceId] : Entrance_Table[entranceId];
+                    string toEntrance = Entrance_Table[read];
+
+                    Console.WriteLine($"{fromEntrance} => {toEntrance}");
```

Comment:
`bqfindindex` is not an adequate replacement for `beta_ent`. 

the purpose of `beta_ent` is to convert a vanilla entranceId into a BetaQuest entranceId that can then be passed into the `ent` command for testing purposes

Second, you shouldn't assume that `read` will always map to Entrance_Table
```suggestion
                    short entranceId = VAR_Entrances[i];
                    string fromEntrance = Grotto_Table.ContainsKey(entranceId) ? Grotto_Table[entranceId] : Entrance_Table[entranceId];
                    string toEntrance = Entrance_Table.ContainsKey(entranceId) ? Entrance_Table[read] : $"{VAR_Entrances[i]:X4}" ;

                    Console.WriteLine($"{fromEntrance} => {toEntrance}");
```